central/policy/service/customresource/testdata/custom_resource.yaml

```diff
@@ -0,0 +1,163 @@
+apiVersion: config.stackrox.io/v1alpha1
+kind: SecurityPolicy
+metadata:
+  name: Vulnerable Container
+spec:
+  description: Alert if the container contains vulnerabilities
+  rationale: This is the rationale
+  remediation: This is the remediation
+  disabled: false
+  categories:
+  - Image Assurance
+  - Privileges Capabilities
+  - Container Configuration
+  lifecycleStages:
+  - BUILD
+  eventSource: NOT_APPLICABLE
+  exclusions:
+  - name: exclusionName1
+    deployment:
+      name: deployment1
+      scope:
+        cluster: cluster1
+        namespace: label1
+        label:
+          key: key1
+          value: value1
+    expiration: "2043-12-20T10:45:23Z"
+  - name: exclusionName2
+    deployment:
+      name: deployment2
+      scope:
+        cluster: cluster2
+        namespace: label2
+        label:
+          key: key2
+          value: value2
+  scope:
+  - cluster: prod cluster
+    namespace: stackrox
+    label:
+      key: com.docker.stack.namespace
+      value: prevent
+  severity: LOW_SEVERITY
+  enforcementActions:
+  - FAIL_BUILD_ENFORCEMENT
+  - KILL_POD_ENFORCEMENT
+  policyVersion: "1.1"
```

Comment:
Just ran an experiment on my PR image which removes this - so we will need to remove this from save as cr output as well. if the incoming yaml has it, CR validation fails (k8s path, not my code)

Reply:
Removed